src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.android.cs

```diff
@@ -81,7 +218,7 @@ public void OnPlaybackParametersChanged(PlaybackParameters? playbackParameters)
 	/// This is part of the <see cref="IPlayer.IListener"/> implementation.
 	/// While this method does not seem to have any references, it's invoked at runtime.
 	/// </remarks>
-	public void OnPlayerStateChanged(bool playWhenReady, int playbackState)
+	public async void OnPlayerStateChanged(bool playWhenReady, int playbackState)
```

Comment:
This method is called by android system, so we need to add an `async void` here, think this as an interface contract that we can't change